docs/workers.md

```diff
@@ -347,6 +368,25 @@ stream_writers:
     events: event_persister1
 ```
 
+Each of the streams have associated endpoints which should have `POST` and `PUT`
+requests routed to the workers handling that stream. Otherwise, those requests
+will be proxied to the proper worker.
```

Comment:
Sadly, I don't think this is correct. Examples:

* if you send a `PUT /typing` request to the wrong worker, it will 500.
* if you send a `PUT /sendToDevice` request or a `POST /receipt` request to the wrong worker... I dunno. I think it will still work? It won't proxy it, anyway.

I think it's roughly correct for `account_data`, `tags` and `presence`

Reply:
That's quite confusing! My initial thought -- is this a bug? Are those supposed to be proxied to the proper workers and we just don't? It would be much clearer to document if the various endpoints behaved the same.

Reply:
yeah. No idea what the intention is here.

Reply:
@erikjohnston Any idea what's *supposed* to happen here?

Reply:
Ideally I think we want requests to be proxied to the right worker, so that the end game supports load balancers just round robining all requests to all workers (even if that is not the most efficient way of doing it).